packages/account-v2/src/modules/IDVForm/index.tsx

```diff
@@ -0,0 +1,159 @@
+import { Field, FieldProps, FormikProps, useFormikContext } from 'formik';
+import { ResidenceList } from '@deriv/api-types';
+import { WalletDropdown } from '../../components/base/WalletDropdown';
+import { WalletTextField } from '../../components/base/WalletTextField';
+import { ResponsiveWrapper } from '../../components/responsive-wrapper';
+import { DOCUMENT_LIST } from '../../mocks/idv-form.mock';
+import { getIDVNotApplicableOption } from '../../utils/default-options';
+
+type TIDVFormProps = {
+    allowIDVSkip?: boolean;
+    selectedCountry: ResidenceList[0];
+};
+
+type TDocument = {
+    additional?: {
+        display_name?: string;
+        example_format?: string;
+    };
+    example_format?: string;
+    id: string;
+    text: string;
+    value?: string;
+};
+
+type TIDVFormValues = {
+    document_additional?: string;
+    document_number: string;
+    document_type: TDocument;
+    error_message?: string;
+};
+
+type TDropDownList = {
+    text: string;
+    value: string;
+};
+
+export const IDVForm = ({ allowIDVSkip, selectedCountry }: TIDVFormProps) => {
+    const { errors, handleChange, setFieldValue, values }: FormikProps<TIDVFormValues> = useFormikContext();
+    const [documentList, setDocumentList] = useState<TDropDownList[]>([]);
+
+    const defaultDocument = {
+        example_format: '',
+        id: '',
+        text: '',
+        value: '',
+    };
+
+    const { documents_supported } = selectedCountry?.identity?.services?.idv ?? {};
+
+    const IDV_NOT_APPLICABLE_OPTION = useMemo(() => getIDVNotApplicableOption(allowIDVSkip), [allowIDVSkip]);
+
+    const bindDocumentData = (item: string) => {
+        setFieldValue('document_type', getSelectedDocumentConfigData(item), true);
+        if (item === IDV_NOT_APPLICABLE_OPTION.value) {
+            setFieldValue('document_number', '', true);
+            setFieldValue('document_additional', '', true);
+        }
+    };
+
+    const handleSelect = (item: string) => {
+        if (item === 'No results found') {
+            setFieldValue('document_type', defaultDocument, true);
+        } else {
+            bindDocumentData(item);
+        }
+    };
+
+    const getSelectedDocumentConfigData = useCallback((item: string) => {
+        return DOCUMENT_LIST.find(doc => doc.id === item);
+    }, []);
+
+    useEffect(() => {
+        if (Object.keys(documents_supported as Exclude<typeof documents_supported, undefined>)?.length) {
+            const docList = Object.keys(documents_supported as Exclude<typeof documents_supported, undefined>).map(
+                (key: string) => {
+                    return {
+                        text: (documents_supported as Exclude<typeof documents_supported, undefined>)[key].display_name,
+                        value: key,
+                    };
+                }
+            );
+            setDocumentList(docList as TDropDownList[]);
+        }
+    }, [documents_supported]);
+
+    return (
+        <Fragment>
```

Comment:
Some places you are using `React.Fragment` and other places you are using `Fragment`
Have a standard convention.

Reply:
Thank you. I will change this